ipyleaflet/leaflet.py

```diff
@@ -2,16 +2,19 @@
 # Distributed under the terms of the Modified BSD License.
 #
 
+from ast import In
```

Comment:
Is this needed?

Reply:
This was here. I just removed it.

Reply:
The diff says it's an addition, not a removal